clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/cache/backend/ObjectCacheConfig.java

```diff
@@ -1,11 +1,13 @@
 package com.linkedin.davinci.store.cache.backend;
 
+import com.linkedin.davinci.client.CacheValueTransformer;
 import java.util.Optional;
 
 
 public class ObjectCacheConfig {
   private Optional<Long> maxCacheSize = Optional.empty();
   private Optional<Long> ttlInMilliseconds = Optional.empty();
```

Comment:
For TTL and Transformer logic, we plan to make it per store config and set it programmatically when user logic is specifying DaVinciConfig.